pom.xml

```diff
@@ -260,19 +260,20 @@
         <version.groovy-all>2.4.7</version.groovy-all>
         <version.httpunit>1.7.2</version.httpunit>
         <version.io.agroal>1.3</version.io.agroal>
-        <version.io.jaegertracing>0.34.0</version.io.jaegertracing>
+        <version.io.jaegertracing>0.35.5</version.io.jaegertracing>
         <version.io.netty>4.1.34.Final</version.io.netty>
-        <version.io.opentracing>0.31.0</version.io.opentracing>
-        <version.io.opentracing.concurrent>0.1.0</version.io.opentracing.concurrent>
-        <version.io.opentracing.jaxrs2>0.1.7</version.io.opentracing.jaxrs2>
-        <version.io.opentracing.tracerresolver>0.1.5</version.io.opentracing.tracerresolver>
-        <version.io.opentracing.servlet>0.1.0</version.io.opentracing.servlet>
+        <version.io.opentracing>0.33.0</version.io.opentracing>
```

Comment:
This is wrong OT version should be 0.31.0 https://github.com/eclipse/microprofile-opentracing/blob/master/tck/base/pom.xml#L83